tests/audit-tests/src/test/java/uk/gov/di/txma/audit/pages/IPVCoreStub/VerifiableCredentialsPage.java

```diff
@@ -16,4 +16,10 @@ public VerifiableCredentialsPage() {
     @FindBy (xpath = "//div[@class='govuk-details__text']//pre")
     public WebElement CRIJSONResponse;
 
+    @FindBy (xpath = "/html/body/div/main/div/div/dl[4]/div/dd/details/summary/span")
+    public WebElement PassportCredential_attributes_link;
+
+    @FindBy (xpath = "//div[@class='govuk-details__text']//pre")
+    public WebElement PassportCRIJSONResponse;
```

Comment:
I don't think this is the correct page for these. Again, they are part of the orchestration journey and the page is different.